duckduckgo/iOS
# macOS: Display error messaging for cancelled subscriptions

Task/Issue URL: https://app.asana.com/0/72649045549333/1205438842252963/f

**Description**:

See macOS PR:
https://github.com/duckduckgo/macos-browser/pull/2394

**Steps to test this PR**:
See macOS PR. For iOS, just check it compiles + perhaps a little smoke test that the VPN works.

**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
